source-code/ide-extension/src/decorations/messagePreview.ts

```diff
@@ -111,4 +113,7 @@ export async function messagePreview(args: { context: vscode.ExtensionContext })
 		undefined,
 		args.context.subscriptions,
 	)
+
+	// update decorations, when message was edited
+	onDidEditMessage(() => updateDecorations());
```

Comment:
Nice! Not listenting to all of the files but only update if we have a change event 👍